### ISSUE TITLE
Python3 error

### ISSUE THREAD

**Issue by ShigekiKarita** [NVIDIA/pynvrtc#1]
str objects should be encoded.

```
EEEEE.
======================================================================
ERROR: test_compile_empty_program (compile-tests.CompileTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/skarita/Desktop/pynvrtc/tests/compile-tests.py", line 39, in test_compile_empty_program
    ['__device__ void f() {}\n'], ['foo.h'])
  File "/home/skarita/Desktop/pynvrtc/pynvrtc/interface.py", line 181, in nvrtcCreateProgram
    headers_array[:] = headers
TypeError: bytes or integer address expected instead of str instance

======================================================================
ERROR: test_create_program (compile-tests.CompileTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/skarita/Desktop/pynvrtc/tests/compile-tests.py", line 31, in test_create_program
    [], [])
  File "/home/skarita/Desktop/pynvrtc/pynvrtc/interface.py", line 185, in nvrtcCreateProgram
    c_char_p(src), c_char_p(name),
TypeError: bytes or integer address expected instead of str instance

======================================================================
ERROR: test_lower_name (compile-tests.CompileTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/skarita/Desktop/pynvrtc/tests/compile-tests.py", line 71, in test_lower_name
    'simple.cu', [], [])
  File "/home/skarita/Desktop/pynvrtc/pynvrtc/interface.py", line 185, in nvrtcCreateProgram
    c_char_p(src), c_char_p(name),
TypeError: bytes or integer address expected instead of str instance

======================================================================
ERROR: test_program_log (compile-tests.CompileTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/skarita/Desktop/pynvrtc/tests/compile-tests.py", line 48, in test_program_log
    [], [])
  File "/home/skarita/Desktop/pynvrtc/pynvrtc/interface.py", line 185, in nvrtcCreateProgram
    c_char_p(src), c_char_p(name),
TypeError: bytes or integer address expected instead of str instance

======================================================================
ERROR: test_program_output (compile-tests.CompileTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/skarita/Desktop/pynvrtc/tests/compile-tests.py", line 60, in test_program_output
    [], [])
  File "/home/skarita/Desktop/pynvrtc/pynvrtc/interface.py", line 185, in nvrtcCreateProgram
    c_char_p(src), c_char_p(name),
TypeError: bytes or integer address expected instead of str instance

----------------------------------------------------------------------
Ran 6 tests in 0.020s

FAILED (errors=5)
WARNING: NVRTC_LIBRARY is not set, assuming it is in [DY]LD_LIBRARY_PATH/PATH
```

**Comment by seanprime7** [comment]
@ShigekiKarita, can this be closed now that PR #2 has been merged?

**Comment by ShigekiKarita** [comment]
@seanprime7  Thanks for merging!
